sql: update admin.md about `show master status` (#683)
pingcap/docs: sql/admin.md

> - To view statistics using the `SHOW` statement, see [View Statistics](../sql/statistics.md#view-statistics).
> - For more information about the `SHOW` statement, see [SHOW Syntax in MySQL](https://dev.mysql.com/doc/refman/5.7/en/show.html).

> In TiDB, the `UniqueID` field returned from `SHOW MASTER STATUS` is actually current `TSO` obtained from `PD`, the `TSO` is used during binlog increment synchronization.

Reviewer: ```suggestion
The `UniqueID` field returned from `SHOW MASTER STATUS` is the current `TSO` obtained directly from `PD`. The `TSO` is used during binlog increment and [`mydumper`](../tools/mydumper.md) synchronization.
```
